ui/src/authors/containers/DetailPage.jsx

```diff
@@ -115,12 +116,19 @@ class DetailPage extends Component {
                     </Col>
                   )}
                   <Col xs={24} md={12} lg={9} xl={8}>
-                    <CitationSummaryContainer
+                    <CitationTableContainer
                       searchQuery={this.authorLiteratureSearchQuery}
```

Comment:
`searchQuery` isn't needed anywhere now, since you're also touching `CitationTable`